### DISCUSSION TITLE
Migration Error - v9.0.0-rc.88 to v9.0.0-rc.100

### DISCUSSION THREAD

**Discussion by sarthakgaur** [directus/directus#9348] [Archive]
I am trying to migrate from v9.0.0-rc.88 to v9.0.0-rc.100, but I am getting the following error when I try to run the migrations:

```
$ npx directus database migrate:latest
18:26:03 ✨ Loaded extensions: link, tags
18:26:03 ✨ Running migrations...
18:26:03 ✨ Applying Add Auth Provider...
18:26:03 🚨 error alter table "directus_users" drop constraint "directus_users_email_unique" - constraint "directus_users_email_unique" of relation "directus_users" does not exist 
error: alter table "directus_users" drop constraint "directus_users_email_unique" - constraint "directus_users_email_unique" of relation "directus_users" does not exist
    at Parser.parseErrorMessage (/home/john/project/node_modules/pg-protocol/dist/parser.js:287:98)
    at Parser.handlePacket (/home/john/project/node_modules/pg-protocol/dist/parser.js:126:29)
    at Parser.parse (/home/john/project/node_modules/pg-protocol/dist/parser.js:39:38)
    at Socket.<anonymous> (/home/john/project/node_modules/pg-protocol/dist/index.js:11:42)
    at Socket.emit (events.js:400:28)
    at addChunk (internal/streams/readable.js:293:12)
    at readableAddChunk (internal/streams/readable.js:267:9)
    at Socket.Readable.push (internal/streams/readable.js:206:10)
    at TCP.onStreamRead (internal/stream_base_commons.js:188:23)

```

Any help will be appreciated! Thanks!

**Comment by narduin** [comment]
I'm having the same issue trying to migrating from rc-94 to rc-100 with a sqlite database.
I'm trying to manually debug this, I'll keep you informed!

Although I think you should create an issue rather than a discussion :)

**Reply by sarthakgaur** [reply]
@narduin Thanks for the suggestion! I'll create an issue on this topic soon.

**Reply by sarthakgaur** [reply]
@narduin My issue is resolved. https://github.com/directus/directus/issues/10899

**Comment by shashwattpandeyy** [comment]
@sarthakgaur did you able to solve this error because I'm facing the same error?

**Reply by sarthakgaur** [reply]
No, but there is something you can try out. Try updating in steps, so instead of directly updating to the latest version, update to the next version and run migrations, and then keep repeating this until you end with the latest version.

So, instead of v9.0.0-rc.88 to v9.0.0-rc.100. Do it like v9.0.0-rc.88 -> v9.0.0-rc.89 -> v9.0.0-rc.90. I was running into some issue while I tried this out that I am not able to recall, but it's something to try out.

Please make a backup of the database before trying this out.

**Reply by sarthakgaur** [reply]
@shashwattpandeyy My issue is resolved. https://github.com/directus/directus/issues/10899

**Comment by sarthakgaur** [comment]
https://github.com/directus/directus/issues/10899
